### DISCUSSION TITLE
Why image captions can not be on the side?

### DISCUSSION THREAD

**Discussion by gguio** [ox-tufte/ox-tufte#29] [General]
I guess it would require changing the HTML structure but what’s wrong about it? I’m just genuinely interested what is the reason for this as captions on the side is one of my favourite features of tufte css

**Comment by suhail-singh** [comment]
> what’s wrong about it?

I'm not sure I understand.  Who said that it's "wrong"?  There's nothing
stopping someone from adding a marginnote that provides a caption for a figure.

https://edwardtufte.github.io/tufte-css/ shows examples of caption being both
underneath a figure as well as separately on the side via a marginnote.  As far
as I'm aware, both are possible to express with `ox-tufte`.

Is there something you're trying to do and it's not working as expected?  If so,
I encourage you to create an issue and provide steps to reproduce.  Code snippet
using `org-export-string-as` showing current HTML output vs desired HTML output
is strongly encouraged.

I am closing this discussion for now, but if what you're experiencing isn't a
bug and my comment above hasn't answered your query, please don't hesitate to
reopen.

**Reply by gguio** [reply]
Thanks, I didn’t think of using margin note this way, it gives me desired result 👌🏻
